shadows/framework/src/main/java/org/robolectric/shadows/ShadowCryptoObject.java

```diff
@@ -0,0 +1,19 @@
+package org.robolectric.shadows;
+
+import static android.os.Build.VERSION_CODES.P;
+
+import android.hardware.biometrics.CryptoObject;
+import org.robolectric.annotation.HiddenApi;
+import org.robolectric.annotation.Implementation;
+import org.robolectric.annotation.Implements;
+
+@SuppressWarnings("UnusedDeclaration")
+@Implements(value = CryptoObject.class, isInAndroidSdk = false, minSdk = P)
+public class ShadowCryptoObject {
+
+  @Implementation
+  @HiddenApi
```

Comment:
nit: can you add a Javadoc comment summarizing why this shadow method is necessary.

Reply:
Addressed here: https://github.com/robolectric/robolectric/pull/8344